docs/open-source-projects.mdx

```diff
@@ -181,7 +181,93 @@ export const projectsData = [
 			{image:'/img/OSS_screnshots/ggoods_3.png', class:'left-screenshot', link:'#screen-ggoods-3', id:'screen-ggoods-3'},
 			{image:'/img/OSS_screnshots/ggoods_4.png', class:'right-screenshot', link:'#screen-ggoods-4', id:'screen-ggoods-4'}
 		]
-	}
+	},
+	{
+		name: 'MyvoteEOS Proxy Website',
+		description : 'The EOS based proxy facilitates the transition into a truly democratic based chain that is free from Pareto control by voting for BP’s that do not engage with direct vote rebate.',
+		href: '#myvoteeos',
+		website: [
+			{ title: 'Sitio Web', link: 'https://myvoteeos.com/home', name: 'MyvoteEOS' }
+		],
+		github: [
+			{link: 'https://github.com/eoscostarica/MyvoteEOS-proxy', image: 'https://cdn-icons-png.flaticon.com/32/25/25231.png'}
+		],
+		logos: [
+    		{image:'https://raw.githubusercontent.com/eoscostarica/MyvoteEOS-proxy/master/docs/my-vote-eos-logo.png', link:'https://myvoteeos.com/home'}
+		],
+		images: [
+			{image:'/img/OSS_screnshots/MyvoteEOS.png', class:'left-screenshot', link:'#screen-MyvoteEOS-1', id:'screen-MyvoteEOS-1'},
+			{image:'/img/OSS_screnshots/MyvoteEOS_2.png', class:'right-screenshot', link:'#screen-MyvoteEOS-2', id:'screen-MyvoteEOS-2'},
+			{image:'/img/OSS_screnshots/MyvoteEOS_3.png', class:'left-screenshot', link:'#screen-MyvoteEOS-3', id:'screen-MyvoteEOS-3'},
+			{image:'/img/OSS_screnshots/MyvoteEOS_4.png', class:'right-screenshot', link:'#screen-MyvoteEOS-4', id:'screen-MyvoteEOS-4'}
+		]
+	},  
+	{
+		name: 'Eden Smart Contracts',
+		description : 'Eden Community Web App and Smart Contracts ',
```

Comment:
'Updates and Maintenance to Eden Community Web App and Smart Contracts'

(remove trailing whitespace)